packages/user/Events/src/REvents.cpp

```diff
@@ -1097,6 +1098,14 @@ std::vector<char> REvents::sqlQuery(const std::string& squery)
          {
             abortMessage(std::string("sqlite3_prepare_v2: ") + sqlite3_errmsg(db));
          }
+         for (std::size_t i = 0; i < params.size(); ++i)
+         {
+            if (int err = sqlite3_bind_text(stmt, static_cast<int>(i + 1), params[i].c_str(),
```

Comment:
Technically, `sqlite3_bind_text64` is safer. The maximum wasm memory can be set to 4GiB, and fracpack also supports up to 4GiB, which is enough to overflow a signed int. Unpacking the strings will create two copies, so anything that overflows an int will fail even then, but I don't like relying on that.

Reply:
Done